### PR TITLE
Garage Limit

### PR DESCRIPTION
**Pull request by DardoTheMaster**
This will allow server admins to limit the total vehicles (For each kind: Air,Car) storable in the garage.Could be integrated with the donor license. Remember as stated above, that this limit is type specific,so it doesn't refer to all vehicles,but only to the specific type.

It may need more testing.

### CODE REVIEW

**Comment by NiiRoZz** [comment]
I think if i pull that you need to add if véhicle limit are -1 so the limit doesn't work

**Comment by setoy** [comment]
Vehicle limit -1 == disabled. And a comment to the variable in config_master.hpp may be needed for that, then.

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -2229,6 +2229,10 @@
       <Portuguese>Não existe um veículo por perto...</Portuguese>
       <Polish>W pobliżu nie ma pojazdu...</Polish>
     </Key>
+    <Key ID="STR_NOTF_TooMuchVehicles">
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: <German>Du kannst nicht mehr als %1 Fahrzeuge einparken</German>

**Comment by DardoTheMaster** [comment]
Should be okay now

**Comment by tkcjesse** [comment]
@DardoTheMaster Do you mind if I redo this as a single pull? Otherwise if you want to feel free. Just would be a lot cleaner.

**Comment by clackd1amond** [comment]
When vehicles_limit is set to false, the player's garage gets wiped. And when you sell an item, it doesn't get deleted from your inventory and you can sell it unlimitedly. Haven't tested it with another values.

**Pull request closed without merging**